common/beerocks/bwl/include/bwl/nl80211_client.h

```diff
@@ -33,6 +55,138 @@ namespace bwl {
 class nl80211_client {
 
 public:
+    /**
+     * @brief Channel information
+     *
+     * Used in sBandInfo structure as each band contains a map of supported channels.
+     *
+     * Information obtained with NL80211_CMD_GET_WIPHY command through a NL80211 socket.
+     * See NL80211_FREQUENCY_ATTR_* in <linux/nl80211.h> for a description of each field.
+     */
+    struct sChannelInfo {
```

Comment:
I realize I'm royally late with giving this feedback, but according to our CODING_STYLE.md, we use lower snake case everywhere (except for cases of historical accident, like sMacAddress). So this should be `struct channel_info`.

(nitpick)

Reply:
Done